### DISCUSSION TITLE
Error

### DISCUSSION THREAD

**Discussion by juanplayerofc** [victornpb/undiscord#469] [4. Bugs and Issues]
Everytime i start deleting, it deletes about 25 messages and gives me this error

[End condition A].	{"running":true,"delCount":0,"failCount":0,"grandTotal":84483,"offset":0,"iterations":1,"_seachResponse":{"total_results":84483,"messages":[],"analytics_id":"869e4e5dfadc87197c4eda4790006df9"},"_messagesToDelete":[],"_skippedMessages":[]

**Comment by victornpb** [comment]
duplicated #471
